pybamm/models/full_battery_models/base_battery_model.py

```diff
@@ -202,8 +209,8 @@ def options(self, extra_options):
         default_options = {
             "operating mode": "current",
             "dimensionality": 0,
-            "surface form": False,
-            "convection": False,
+            "surface form": "false",
+            "convection": "none",
```

Comment:
Linked to my previous comment, how come here we use `"none"` instead of `"false"`?

Reply:
It is a bit arbitrary but I am basically using "false" if the other option is "true" and "none" otherwise, except for "surface form". "There is no convection" -> "convection: none" vs "We don't use the surface form" -> "surface form: false"